server/src/test/java/org/elasticsearch/gateway/DanglingIndicesStateTests.java

```diff
@@ -142,7 +172,7 @@ public void testDanglingProcessing() throws Exception {
 
             // check that several runs when in the metadata, but not cleaned yet, still keeps dangled
             for (int i = 0; i < numberOfChecks; i++) {
-                Map<Index, IndexMetadata> newDanglingIndices = danglingState.findNewDanglingIndices(metadata);
+                Map<Index, IndexMetadata> newDanglingIndices = danglingState.findNewDanglingIndices(emptyMap(), metadata);
```

Comment:
Similarly here, we should be using the previous result rather than `emptyMap()`.